src/integrationTest/sam.test.ts

```diff
@@ -120,16 +116,13 @@ async function closeAllEditors(): Promise<void> {
     await vscode.commands.executeCommand('workbench.action.closeAllEditors')
 }
 
-function validateLocalInvokeResult(
-    actualResult: LocalInvokeCodeLensCommandResult,
-    expectedResult: LocalInvokeCodeLensCommandResult
-) {
-    assert.strictEqual(actualResult.datum.name, expectedResult.datum.name)
-    assert.strictEqual(actualResult.datum.value, expectedResult.datum.value)
-    assert.strictEqual(actualResult.datum.unit, expectedResult.datum.unit)
+function validateLocalInvokeResult(actualResult: MetricDatum, expectedResult: MetricDatum) {
```

Comment:
This could probably be renamed to something like assertMetricDatumMatch

Reply:
This is actually testing the invoke result is correct with the side effect of it using metricdatum